Content.Server/Chemistry/EntitySystems/SolutionPurgeSystem.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Server.Chemistry.Components;
+using Content.Server.Chemistry.Components.SolutionManager;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Timing;
+
+namespace Content.Server.Chemistry.EntitySystems;
+
+public sealed class SolutionPurgeSystem : EntitySystem
+{
+    [Dependency] private readonly SolutionContainerSystem _solutionContainer = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<SolutionPurgeComponent, EntityUnpausedEvent>(OnUnpaused);
+    }
```

Comment:
Needs to update nextpurge on mapinit if it's < curtime.

Reply:
I believe I have accomplished this in a2bffd7. Let me know if I missed something!